src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -50,7 +61,27 @@ function NameAndPicture({ navigation, route }: Props) {
   const asyncKomenciReadiness = useAsyncKomenciReadiness()
   const showGuidedOnboarding = useSelector(showGuidedOnboardingSelector)
   const createAccountCopyTestType = useSelector(createAccountCopyTestTypeSelector)
-  const skipUsername = route.params?.skipUsername //TODO repalce with statsig variable
+
+  useEffect(() => {
```

Comment:
is useEffect the right hook to use here? My understanding is that useEffect is for a different use case, where you want to run some side-effect just after the render phase.

Reply:
maybe getting rid of useState and useEffect, and using useCallback instead would be better. But even that feels like overkill; observing experiment parameters shouldn't be expensive to do on each render.